pipeline/acquisition.py

```diff
@@ -0,0 +1,362 @@
+        ['behavior'], ['extracelluar'], ['photostim']
+    ]
+
+@schema 
+class ActionLocation(dj.Manual): 
+    definition = """ # Information relating the location of any experimental task (e.g. recording (extra/intra cellular), stimulation (photo or current) )
+    -> reference.BrainLocation
+    -> reference.CoordinateReference
+    coordinate_ap: float    # in mm, anterior positive, posterior negative 
+    coordinate_ml: float    # in mm, always postive, number larger when more lateral
+    coordinate_dv: float    # in mm, always postive, number larger when more ventral (deeper)
+    """
+    
+#@schema
+#class PhotoStim(dj.Manual):
+#    definition = """
+#    photo_stim_id: int
+#    ---
+#    photo_stim_wavelength: int
+#    photo_stim_method: enum('fiber', 'laser')
+#    -> reference.BrainLocation.proj(photo_stim_location="brain_location")
+#    -> reference.Hemisphere.proj(photo_stim_hemisphere="hemisphere")
+#    -> reference.CoordinateReference.proj(photo_stim_coordinate_ref="coordinate_ref")
+#    photo_stim_coordinate_ap: float    # in mm, anterior positive, posterior negative 
+#    photo_stim_coordinate_ml: float    # in mm, always postive, number larger when more lateral
+#    photo_stim_coordinate_dv: float    # in mm, always postive, number larger when more ventral (deeper)
+#    """
+
+@schema
+class Session(dj.Manual):
+    definition = """
+    -> subject.Subject
+    session_time: datetime    # session time
+    ---
+    session_directory = "": varchar(256)
+    session_note = "" : varchar(256) 
+    """
+
+    class Experimenter(dj.Part):
+        definition = """
+        -> master
+        -> reference.Experimenter
+        """
+
+    class ExperimentType(dj.Part):
+        definition = """
+        -> master
+        -> ExperimentType
+        """
+
+@schema
+class IntracellularInfo(dj.Manual):
+    definition = """ # Table containing information relating to the intracelluar recording (e.g. cell info)
+    -> Session
+    cell_id: varchar(64)
+    ---
+    cell_type: enum('excitatory','inhibitory','N/A')
+    -> ActionLocation
+    -> reference.Device
+    """    
+    
+@schema
+class ExtracellularInfo(dj.Manual):
+    definition = """ # Table containing information relating to the extracelluar recording (e.g. location)
+    -> Session
+    ec_id: varchar(64)
+    ---
+    -> ActionLocation
+    -> reference.Device
+    """    
+    
+@schema
+class StimulationInfo(dj.Manual):
+    definition = """ # Table containing information relating to the stimulatiom (stimulation type (optical or electrical), location, device)
+    -> Session
+    stim_id: varchar(64)
+    ---
+    stim_type: enum('optical','electrical')
+    -> ActionLocation
+    -> reference.Device
+    """    
+
+@schema
+class TrialSet(dj.Imported):
+    definition = """
+    -> Session
+    ---
+    n_trials: int # total number of trials
+    """
+    class Trial(dj.Part):
+        definition = """
+        -> master
+        trial_id: varchar(32)
```

Comment:
Wht's a `trial_id`.  Please comment for clarity.  Can a smaller datatype be used?